src/main/java/com/jland/repository/impl/ConferenceRepositoryImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.jland.repository.impl;
```

Comment:
in your case it should be .repository.jdbc

Reply:
changed